bundles/infix-en_GB-jdk8/atrium-infix-en_GB-jdk8-jvm/src/test/kotlin/custom/SmokeSpec.kt

```diff
@@ -0,0 +1,45 @@
+package custom
+
+import ch.tutteli.atrium.api.verbs.assertThat
+import ch.tutteli.atrium.assertions.Assertion
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+import ch.tutteli.atrium.reporting.RawString
+import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
+import ch.tutteli.atrium.translations.DescriptionBasic
+import org.jetbrains.spek.api.Spek
+import java.nio.file.Paths
+
+object SmokeSpec : Spek({
+    test("see if `toBe` can be used") {
+        // TODO #233 uncomment once toBe is implemented
+        //assertThat(1) toBe 2
+    }
+
+    test("see if `Path.existsNot` can be used") {
+        assertThat(Paths.get("nonExisting"))
+    }
+
+    test("see if own assertion function without i18n can be used") {
+        // TODO #233 uncomment once toBe is implemented
+        //assertThat(2) toBe even
+    }
+
+    test("see if own assertion function with i18n can be used") {
+        assertThat(4).isMultipleOf(2)
+    }
+})
+
+fun Expect<Int>.isEven(): Expect<Int> =
```

Comment:
Please implement it as infix function, see SmokeSpec in atrium-cc-infix-en_GB-robstoll for an example